src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -62,23 +64,32 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
                       protected val authProvider: LeoAuthProvider,
                       protected val serviceAccountProvider: ServiceAccountProvider)
                      (implicit val executionContext: ExecutionContext) extends LazyLogging {
+
   private val bucketPathMaxLength = 1024
   private val includeDeletedKey = "includeDeleted"
 
+  val config = ConfigFactory.parseResources("reference.conf").withFallback(ConfigFactory.load())
```

Comment:
I think we can just remove `isWhitelisted` and this `config` object, right? The `/isWhitelisted` was just a temporary hack anyway.

Reply:
I was told by Brad that we have to keep the whitelist endpoint - as the FC Notebooks UI still has to be behind a whitelist until after the redesign

Reply:
Oh :( ok nevermind then.